stub/UVFunctions.php

```diff
@@ -1840,6 +1819,97 @@ function uv_fs_event_init(UVLoop $loop, string $path, callable $callback, int $f
 {
 }
 
+//
+// still needs more checking and some not part of `libuv`
+//
+
+/**
+ * @param UVLoop|null $uv_loop
+ *
+ * @return void
+ */
+function uv_run_once(UVLoop $uv_loop = null)
+{
+}
+
+/**
+ * Returns current uv type. (this is not libuv function. util for php-uv).
+ *
+ * @param resource $uv uv_handle.
+ *
+ * @return int  should return UV::IS_UV_* constatns. e.g) UV::IS_UV_TCP.
+ */
+function uv_handle_type($uv)
+{
+}
+
+/**
+ * @param UVLoop $loop
+ * @param array $options
+ * @param int $optmask
+ *
+ * @return resource
+ */
+function uv_ares_init_options(UVLoop $loop, array $options, int $optmask)
+{
+}
+
+/**
+ * @param resource $handle
+ * @param string $name
+ * @param int $flag
+ * @param callable $callback
+ *
+ * @return void
+ */
+function ares_gethostbyname($handle, string $name, int $flag, callable $callback)
```

Comment:
Where do you get these areas functions from?

Reply:
Not sure now, i thought from readme or from stub i pulled in from phpstorm, i will remove.